api/game_test.go

```diff
@@ -27,15 +27,14 @@ var _ = Describe("Game Handler", func() {
 		recorder = httptest.NewRecorder()
 	})
 
-	Describe("PUT /games", func() {
+	Describe("PUT /games/{id}", func() {
 		It("should return status code of 200", func() {
 			id := uuid.NewV4().String()
 			gameReader := JSONFor(JSON{
-				"ID":       id,
 				"Name":     "Game Awesome Name",
 				"BundleID": "com.topfreegames.example",
 			})
-			request, _ := http.NewRequest("PUT", "/games", gameReader)
+			request, _ := http.NewRequest("PUT", "/games/"+id, gameReader)
```

Comment:
I didn't even know we could add strings in go. I'd use Sprintf for this:

`fmt.Sprintf("/games/%s", id)`